src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.Common/README.md

```diff
@@ -1,6 +1,22 @@
 # Microsoft.Extensions.Diagnostics.HealthChecks.Common
 
-Health check implementations.
+This package contains common health check implementations. Here are the main features it provides:
+
+Application Lifecycle Health Check
+- A provider that's tied to the application's lifecycle based on `IHostApplicationLifetime`
+- Emits unhealthy status when the application is not started, stopping, or stopped
+- Emits healthy status when the application is started and running
+
+Manual Health Check
+- A provider that enables manual control of the application's health
+- Emits unhealthy status when `ReportUnhealthy` is called
+- Emits healthy status when `ReportHealthy` is called
+
+Telemetry Publisher
+- A publisher which emits telemetry (logs and counters) representing the application's health
+- Can be configured to log only when unhealthy reports are received
```

Comment:
Is there any doc we can link to for the counters/tags? Or should we mention those explicitly here:

**dotnet.health_check.reports**

- dotnet.health_check.status

**dotnet.health_check.unhealthy_checks**

- dotnet.health_check.name
- dotnet.health_check.status

Reply:
I'm not sure we need these details here 🤔

Reply:
If you really want to have this reference, we can add a link to documentation that @iliar-turdushev prepared and sent to David Pine

Reply:
Done